source/_usecases/find the Index of an element in an Array

```diff
@@ -0,0 +1,17 @@
+extends: _layouts.usecase
+date: 2018-03-27
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex
+reference: find the Index of an element in an Array
+related: find-the-Index-of-an-element-in-an-Array 
+category: arrays
+
+```javascript
+var colors = ["red", "green", "blue"];
+colors.indexOf("blue");
```

Comment:
let's remove the 1st and 3rd example, and only keep the 2nd one:
`colors.indexOf("green");`

and then only one `<pre class="output">1</pre>`
Thanks!

Reply:
@jadjoubran ok thanks man!